src/js/classes/crop-area-circle.js

```diff
@@ -1,154 +0,0 @@
-(function () {
```

Comment:
Then to keep it consistent:


- [ ] Project description is wrong.
> Image Editor directive for AngularJS. Enables to crop a **circle** or a square out of an image.

- [ ] First screenshot example is incorrect: https://github.com/partnermarketing/pm-image-editor#screenshots



- [ ] Options are not correct, need to be updated to the ones we have now.
https://github.com/partnermarketing/pm-image-editor#options


- [ ] References to circle i here as well:
https://github.com/partnermarketing/pm-image-editor#result-image

- [ ] Non existing config: https://github.com/partnermarketing/pm-image-editor#area-type